lib/common/test/unit-tests/helpers.ts

```diff
@@ -699,4 +707,120 @@ describe("helpers", () => {
 			});
 		});
 	});
+
+	const setNpmConfigArgv = (original: string[]): void => {
+		process.env.npm_config_argv = JSON.stringify({ original });
+	};
+
+	describe("doesCurrentNpmCommandMatch", () => {
+		[
+			{
+				name: "returns true when searching for --global flag and it is passed on terminal",
+				input: ["install", "--global", "nativescript"],
+				expectedOutput: true
+			},
+			{
+				name: "returns true when searching for -g flag and --global is passed on terminal",
```

Comment:
this case is passing `-g`, not `--global`, also, as far as I see we are always searching for both of the global flags in each of these cases.

Reply:
done